libs/core/src/lib/breadcrumb/breadcrumb-item.component.ts

```diff
@@ -51,7 +51,7 @@ export class BreadcrumbItemComponent implements AfterViewInit {
     constructor(public readonly elementRef: ElementRef<HTMLElement>) {}
 
     /** @hidden */
-    get _needsClickProxy(): boolean {
+    get needsClickProxy(): boolean {
```

Comment:
why is a getter annotated by `@hidden`. If you remove the `_` prefix you should also remove the `@hidden` typedocs and provide some description for the API docs

Reply:
I should not have removed the `_`, forgot that we use that for hidden vars/functions (in addition to private vars/functions).

I assume it's a getter due to the expression:
```
    /** @hidden */
    get _needsClickProxy(): boolean {
        return (
            !!this.breadcrumbLink?.elementRef().nativeElement.getAttribute('href') || !!this.breadcrumbLink.routerLink
        );
    }
```